frontend/app/views/joining/tierChooser.scala.html

```diff
@@ -4,71 +4,45 @@
-                        <div class="event-content__second">
-                            @if(event.venue.name){
-                                <span class="event-item__location-name">@event.venue.name</span>
-                            }
-                            @for(location <- event.venue.address) {
-                                <span class="event-item__location-detail">@if(event.venue.name){, } @location.city, @location.postal_code</span>
-                            }
-                        </div>
-                    </div>
-
-                    <div class="event-item__price">
-                        @for(ticket <- event.generalReleaseTicket) {
-                            @for(pricing <- ticket.cost) {
-                                <div class="event-content__price event-content--line-divider">@pricing.formattedPrice</div>
-                                <div class="event-content__trail">
-                                    <div class="event-item__price-amount-discount">Partners/Patrons: @pricing.discountPrice</div>
-                                </div>
-                            }
-                        }
-                    </div>
-                </div>
-            }
-            </section>
-        </div>
-    </section>
+@main("Join Choose Tier", pageInfo=pageInfo) {
+
+    <main role="main" class="page-content">
+        @fragments.page.pageHeader("Guardian Live events are exclusively for Guardian members")
+
+        <section class="page-section page-section--no-padding">
+            <div class="page-section__lead-in">
+                @fragments.joiner.joinStepCounter(1, 3)
+            </div>
+            <div class="page-section__content page-section__content--full">
+                <h2 class="h-section h-section--lead">Choose a membership tier to continue with your booking</h2>
+                <ul class="grid grid--slim grid--bordered grid--3up-stacked">
+                    <li class="grid__item">
+                        @fragments.tier.packagePromo(Tier.Friend)
+                    </li>
+                    <li class="grid__item">
+                        @fragments.tier.packagePromo(Tier.Partner)
+                    </li>
+                    <li class="grid__item">
+                        @fragments.tier.packagePromo(Tier.Patron)
+                    </li>
+                </ul>
+            </div>
+            <div class="page-section__supplementary">
+                @for(event <- eventOpt) {
+                    <h4 class="h-subsection">You're buying tickets for:</h4>
+                    @fragments.event.itemSnapshot(event)
+                }
+            </div>
+        </section>
+
+        <section class="page-section">
+            <div class="page-section__content">
+                <h3 class="h-section">What's included</h3>
+                @fragments.tier.packageStack(Tier.Friend, "Benefits")
+                @fragments.tier.packageStack(Tier.Partner, "All Friend benefits, plus")
```

Comment:
I'd quite like these "benefits+" captions to come from a config, or maybe a property on `Tier`. Thoughts appreciated.